cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -401,10 +404,26 @@ protected List<EnvVar> getEnvVars() {
 
         varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_BOOTSTRAP_SERVERS, bootstrapServers));
         varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_ADMIN_CLIENT_CONFIG, kafkaBridgeAdminClient == null ? "" : new KafkaBridgeAdminClientConfiguration(reconciliation, kafkaBridgeAdminClient.getConfig().entrySet()).getConfiguration()));
-        varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_CONSUMER_CONFIG, kafkaBridgeConsumer == null ? "" : new KafkaBridgeConsumerConfiguration(reconciliation, kafkaBridgeConsumer.getConfig().entrySet()).getConfiguration()));
-        varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_PRODUCER_CONFIG, kafkaBridgeProducer == null ? "" : new KafkaBridgeProducerConfiguration(reconciliation, kafkaBridgeProducer.getConfig().entrySet()).getConfiguration()));
         varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_ID, cluster));
 
+        if (kafkaBridgeConsumer != null) {
```

Comment:
What happens when the consumer will not be set? It seems like it would cause an _error_ state where things will fail because the env vars will be missing?

Reply:
@scholzj initially I did not had these two statements here, as they're both default initiated in `KafkaBridgeSpec`, but SpotBugs failed and pointed out this.

Reply:
Well, as Paolo already commented on the other part ... I would:
* Not initialize the objects, that can cause all kind of problems down the road
* Simply handle it here with if-else ... set the default end vars if it is null, use the real values if it isn't

Reply:
yeah, +1 with Jakub's suggestion.

Reply:
I've removed the initialisation and keep the if-else statements